metatron/src/main/java/uk/gov/ida/eidas/metatron/MetricsUtils.java

```diff
@@ -0,0 +1,5 @@
+package uk.gov.ida.eidas.metatron;
+
+public class MetricsUtils {
```

Comment:
I can't find this class being used anywhere - is it needed for some sort of prometheus magic?

Reply:
Again, this is now redundant.  But well done on spotting things Codacy doesn't.  Take that as you will ;)